plugins/modules/aci_rest.py

```diff
@@ -447,7 +463,16 @@ def main():
         output_path = aci.params.get("output_path")
         if output_path is not None:
             with open(output_path, "a") as output_file:
-                json.dump([payload_output_file], output_file)
+                if rest_type == "json":
+                    json.dump([payload_output_file], output_file)
+                    output_file.write("\n")
```

Comment:
Is this \n added in the other output_path cases? If not, we need to check if it will be properly handled by the NAE/NDI script that read this file.